common.sh

```diff
@@ -14,7 +14,7 @@ declare -A jdk_latest=(
 	["eclipse-temurin"]="17"
 	["ibmjava"]="8"
 	["ibm-semeru"]=""
-	["amazoncorretto"]="11"
```

Comment:
I'm leaving latest as it should be the version of the parent latest image not of the latest java version (which is still 8 for whatever reasons)